_data/projects.yml

```diff
@@ -97,82 +98,60 @@ projects</a>, including <a href=\"https://www.vmware.com/\">VMware</a>,
       image: "img/projects/uptane_diagram.png"
       status: *adopted
       site: "https://uptane.github.io/"
-      description: "Cars today use a truly enormous
-amount of software, and, like any
-software, it contains bugs.  Unfortunately, there is no secure way
-to update software in cars.  Our Uptane project provides a
-mechanism to securely distribute updates to cars.  Uptane can counter a
-comprehensive array of security attacks, and is resilient to partial
-compromises, while addressing  automotive specific vulnerabilities and
-limitations. Uptane was recently named one of
-the <a href=\"https://www.popsci.com/top-security-innovations-2017#page-2\">Top
-Security Innovations of 2017</a> by Popular Science Magazine. "
-      products: "Uptane has already been adopted by
-<a href=\"/blog/2017-04-04-uptane-blog-post\">multiple auto makers</a>.
-Uptane has been 
-<a href=\"https://advancedtelematic.com/en/press-releases/ats-is-integrating-the-uptane-security-framework-for-over-the-air-software-updates-to-connected-vehicles.html\">integrated</a>
-into multiple products including OTA Plus and ATS Garage, two over-the-air 
-software update products from Advanced Telematic Systems.   ATS also integrated 
-<a href=\"https://github.com/advancedtelematic/aktualizr\">aktualizr</a>, 
-a C++ implementation of Uptane, into <a 
-href=\"https://www.automotivelinux.org/\">Automotive Grade Linux</a>.
-On January 25, 2018, Airbiquity
-announced receipt of a <a href=\"https://www.airbiquity.com/news/press-releases/airbiquity-otamatic-named-2017-new-product-year-business-intelligence-group\">BIG Award for Business</a>
+      description: "Uptane, which applies the basic TUF design to securing software delivered over-the-air to the
+  computerized units of automobiles, can thwart attacks from malicious actors at the manufacturing level. It addresses
```

Comment:
Not really securing from issues at the manufacturing level.  That would be more in-toto or similar.  We're really focused on attacks on the software update infrastructure (which is different from the software development infrastructure).

Reply:
Fixed.